jekyll/_cci2/api-developers-guide.md

```diff
@@ -327,9 +323,9 @@ Notice in the example above that you will receive very specific information abou
 ### Get job details
 {: #get-job-details }
 
-Much like the Get Project Details API request described in the previous example, the Get Job Details API request enables you to return specific job information from the CircleCI API by making a single API request. Retrieving job information can be very useful when you want information about how your job performed, what resources were used (e.g. pipeline, executor type, etc.), and the time it took for the job to finish.
+Much like the Get Project Details API request described in the previous example, the Get Job Details API request enables you to return specific job information from the CircleCI API by making a single API request. 
```

Comment:
The "Get project details" endpoint was either mistyped or has changed -- it's now called "Get a project".
```suggestion
Much like the Get a project API request described in the previous example, the [Get job details](https://circleci.com/docs/api/v2/index.html#operation/getJobDetails) API request enables you to return specific job information from the CircleCI API by making a single API request. 
```